keyboards/crkbd/keymaps/crkdves/glcdfont.c

```diff
@@ -0,0 +1,232 @@
+#include "progmem.h"
+
+// Corne 8x6 font with QMK Firmware Logo
+// Online editor: https://helixfonteditor.netlify.com/
+// See also: https://github.com/soundmonster/glcdfont_converter
+
```

Comment:
/* Copyright 2022 Yeimi Alberth (@yeimialberth)
*  This program is free software: you can redistribute it and/or modify it in the hope that it will be useful.
*/